server/public/styles.css

```diff
@@ -75,10 +75,94 @@ body {
   text-align: center;
   background-color: #ffff64;
   overflow: hidden;
+  display: flex;
+  justify-content: center;
+}
+
+.yellow-contents {
+  width: calc(100% / 3 * 2);
+  margin-bottom: 2.5rem;
+}
+
+.session-data {
+  width: 100vw;
+  display: flex;
+  flex-direction: column;
+}
+
+.session-data > div {
+  margin-left: 1rem;
+  margin-right: 1rem;
+}
+
+.session-stats > div {
+  display: flex;
+  justify-content: center;
+}
+
+.session-stats > div > p {
+  width: 8rem;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.session-times {
+  margin-top: 2rem;
+}
+
+.time-delete {
+  color: #ff4646;
+  display: flex;
+  align-items: center;
+}
+
+.times-list > div {
+  display: flex;
+  justify-content: center;
+}
+
+.times-list > div > * {
+  margin: 0 1rem 0 1rem;
+}
+
+.times-list > div > p:nth-child(1) {
+  width: 2.5rem;
+  text-align: right;
+}
+
+.times-list > div > i:nth-child(3) {
+  width: 2.5rem;
+  text-align: left;
+}
+
+.session-stats > div > p:nth-child(1) {
+  text-align: right;
+  margin-right: 0.25rem;
+}
+
+.session-stats > div > p:nth-child(2) {
+  text-align: left;
+  margin-left: 0.25rem;
```

Comment:
See if you can name these things and use proper CSS classes. These kinds of selectors are very brittle. Also see Bootstrap's text alignment and spacing helper classes.